src/components/ha-gauge.ts

```diff
@@ -88,7 +90,7 @@ export class Gauge extends LitElement {
       </svg>
       <svg class="text">
         <text class="value-text">
-          ${this.value} ${this.label}
+          ${this.valueFormatted} ${this.label}
```

Comment:
This is not different from what it was? And it won't be updated?

Reply:
@bramkragten  `this.valueFormatted` is the value after `formatNumber` has been applied. It is explicitly passed to `ha-gauge` from `hui-gauge-card` with the new state value so `this.valueFormatted` does update when the entity `state` updates. However I can apply `formatNumber` here instead as requested in your other comment.